perf/CMakeLists.txt

```diff
@@ -30,8 +53,17 @@ create_perf_test(PREFIX tuple
                  SOURCES tuple.cc ${PROJECT_SOURCE_DIR}/test/unit/box_test_utils.c
                  LIBRARIES core box tuple benchmark::benchmark
 )
+create_perf_test_target(PREFIX tuple)
```

Comment:
Well, it's not the `PREFIX` but rather `TARGET`, isn't it? If so, should the name of the parameter be adjusted?

Reply:
Renamed.